packages/cli/src/blueprints/.eslintrc.cjs

```diff
@@ -32,11 +29,19 @@
     // TypeScript files
     {
       files: ['**/*.{cts,ts}'],
-      extends: ['plugin:@typescript-eslint/recommended-type-checked'],
+      parser: '@typescript-eslint/parser',
+      extends: [
+        'plugin:@typescript-eslint/recommended',
```

Comment:
I "downgraded" `recommended-type-checked` to `recommended`, as the former would create additional lint errors for `recast` and `ember-template-recast`. At the moment, I don't have the bandwidth to somehow wrap these libraries to fix the lint errors.